frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -138,6 +139,16 @@ export const createDeploymentConfig = (
               image: `${name}:latest`,
               ports,
               env,
+              resources: {
```

Comment:
Does this mean I'm forced to enter resource limits? They shouldn't be required.

Reply:
No, resources is not the required field. resources will be an empty object if the user doesn't enter any value.